### PR TITLE
My first pull request

### PR DESCRIPTION
**Pull request by JuanDeTeo**
My first pull request..

### CODE REVIEW

File: src/app/globals.css [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+      rgba(1, 65, 255, 0),
+      rgba(1, 65, 255, 0.3)
+    );
+
+    --tile-start-rgb: 2, 13, 46;
+    --tile-end-rgb: 2, 5, 19;
+    --tile-border: conic-gradient(
+      #ffffff80,
+      #ffffff40,
+      #ffffff30,
```
Comment (Grekz) [inline src/app/globals.css:62]: **question:** is this the right color for this gradient?

File: src/app/layout.tsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+export const metadata: Metadata = {
+  title: 'Create Next App',
+  description: 'Generated by create next app',
+}
+
+export default function RootLayout({
+  children,
+}: {
+  children: React.ReactNode
```
Comment (Grekz) [inline src/app/layout.tsx:15]: **nitpicking:** I'd prefer that we have the types outside the definition of the component :)

**Review by Grekz — APPROVED** [review]
LGTM 🚀

File: src/app/globals.css (hunk 1 above)
Comment (JuanDeTeo) [reply src/app/globals.css:62]: yes, its a rigth gradient

**Pull request merged**